src/lib/schedule.c

```diff
@@ -89,3 +89,16 @@ int schedule_sof_task_cancel(struct sof_task *task)
 
 	return ret;
 }
+
+int schedule_sof_init(void)
+{
+	int ret = 0;
+
```

Comment:
No need for variable ret.

Reply:
@dbaluta I want to go through all init metods, so I introduce ret variable here. If I made just:
```
return schedulers[i]->scheduler_init();
```
I would invoke only one init function.